test/functional/wallet_orphanedreward.py

```diff
@@ -32,28 +32,31 @@ def run_test(self):
         self.generate(self.nodes[0], 150)
         assert_equal(self.nodes[1].getbalance(), 10 + 25)
```

Comment:
Maybe I am dumb and missing something very obvious, why is the block reward 25 here?

Reply:
Regtest's halving interval is 150 blocks.

Reply:
Ohh.. My bad..